core-api/src/test/java/com/optimizely/ab/OptimizelyTest.java

```diff
@@ -2636,6 +2638,153 @@ public void getVariationWithInvalidDatafile() throws Exception {
 
     //======== Notification listeners ========//
 
+    Boolean isListenerCalled = false;
```

Comment:
Similar comment in https://github.com/optimizely/java-sdk/pull/271 as to how to properly assert something like this.